packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -202,10 +202,7 @@ export class GarbageCollector implements IGarbageCollector {
 					// in the snapshot cannot be interpreted correctly. Set everything to undefined except for
 					// deletedNodes because irrespective of GC versions, these nodes have been deleted and cannot be
 					// brought back. The deletedNodes info is needed to identify when these nodes are used.
-					if (
-						this.configs.gcVersionInBaseSnapshot !==
-						this.summaryStateTracker.currentGCVersion
```

Comment:
I think you can make this private now and use the one from `this.configs` throughout this file.

Reply:
done